README.md

```diff
@@ -24,7 +24,8 @@ This software package is designed for preparing data that can be used to train g
 - [Introduction](#introduction)
 - [Input format](#input-format)
 - [End to end data preparation](#end-to-end-data-preparation)
-    - [Input Flags](#flags)
+    - [Key Flags](#key-flags)
```

Comment:
Should we change this term from key-flags to important-flags. The term key might be overloaded as it is a common Computer science term.

Reply:
Good call, will change!